### ISSUE TITLE
CLDR issue with Date Picker and Data Range Picker

### ISSUE THREAD

**Issue by pa-sen** [SAP/ui5-webcomponents#2237]
**Describe the bug**
When clicking on Date Picker component to choose dates, panel with dates is not displayed and an error occurs in the console:
![image](https://user-images.githubusercontent.com/71762660/94001927-df87c380-fda1-11ea-9dcf-e110b3070c46.png)

**Context**
Was tested with 1.0.0-rc.6 and 1.0.0-rc.8 Web Components version
Angular version 8.2.14.

**Comment by ilhan007** [comment]
Hello @pa-sen 

The CLDR data is one of the assets, used by the ui5 web components. You can read more here: 
https://github.com/SAP/ui5-webcomponents/blob/master/docs/Assets.md


Do you have the following import, it enables the CLDR data

```js
import "@ui5/webcomponents-localization/dist/Assets.js"
```

or this import - it includes CLDR, i18n and theming assets

```js
import "@ui5/webcomponents/dist/Assets.js"
```

You can also read how to build those assets efficiently:
https://github.com/SAP/ui5-webcomponents/blob/master/docs/Assets.md#efficient-asset-bundling

BR,
ilhan

**Comment by SenPavel** [comment]
Hi @ilhan007 ,

Unfortunately these imports do not help, I am getting the same error.

Best regards,
Pavel

**Comment by pa-sen** [comment]
Hi @ilhan007 ,

Could there be any other problems? These import didn't help.

Best regards,
Pavel

**Comment by kmariappan** [comment]
Hi @pa-sen 

I got the similar problem for the datepicker, daterange-picker component.

<img width="599" alt="Screenshot 2020-10-08 at 16 13 57" src="https://user-images.githubusercontent.com/13287428/95478398-c37f4700-0989-11eb-928c-da89673b9a16.png">
<img width="547" alt="Screenshot 2020-10-08 at 16 14 54" src="https://user-images.githubusercontent.com/13287428/95478463-d4c85380-0989-11eb-86b8-479618931802.png">

- I have updated to 1.0.0-rc.9
- Removed the node_modules folder and install the packages( I have updated all other dependencies in my Project)

And it works fine for me.
<img width="411" alt="Screenshot 2020-10-08 at 17 25 38" src="https://user-images.githubusercontent.com/13287428/95479942-9df33d00-098b-11eb-8e31-1e169b51cbc3.png">

I am using VueJs v2.6.11 and node v14.7.0. 

I hope this info helps you.

Best Regards,
Karthik
